### PR TITLE
Update audits to handle limited ads serving

### PR DESCRIPTION
**Pull request by warrengm**
Update audits and docs to handle limited ads: https://developers.google.com/doubleclick-gpt/guides/general-best-practices

Notably this tweeks the `loads-gpt-from-sgdn` audit to be a `loads-gpt-from-official-source`. I left the old docs behind, but we should delete it after launching this audit to our server.

### CODE REVIEW

File: lighthouse-plugin-publisher-ads/test/smoke/expectations/limited-ads-dynamic-loading.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
```
Comment (jburger424) [inline lighthouse-plugin-publisher-ads/test/smoke/expectations/limited-ads-dynamic-loading.js:1]: s/19/20, below too

**Review by jburger424 — APPROVED** [review]

File: docs/audits/loads-gpt-from-official-source.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+from the recommended host. The browser must open a connection to each new host
+your page references, which can take significant time. By loading GPT from the
+recommended host you ensure that all ad serving requests are made to a single
+domain, reducing the number of connections to open and improving page load
+performance.
+
+## Recommendations
+
+Ensure that GPT is being loaded from an official source. For example, standard
+integrations could use
```
Comment (jimper) [inline docs/audits/loads-gpt-from-official-source.md:15]: Standard integrations should use:

File: docs/audits/loads-gpt-from-official-source.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+```HTML
+<script async src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"></script>
+```
+
+And integrations with limited ads could use:
```
Comment (jimper) [inline docs/audits/loads-gpt-from-official-source.md:21]: Integrations that request [limited ads](https://support.google.com/admanager/answer/9882911) should use:

File: lighthouse-plugin-publisher-ads/messages/en-US.json [hunk 4]
```diff
@@ -427,16 +427,16 @@
     "message": "Ad tag is loaded over HTTPS",
     "description": ""
   },
-  "audits/loads-gpt-from-sgdn.js | description": {
-    "message": "Load GPT from 'securepubads.g.doubleclick.net' to reduce GPT load time. By loading GPT from the same host as ad requests, browsers can avoid an additional DNS lookup and HTTP connection. Example:`<script async src=\"https://securepubads.g.doubleclick.net/tag/js/gpt.js\">`. [Learn more](https://developers.google.com/publisher-ads-audits/reference/audits/loads-gpt-from-sgdn).",
+  "audits/loads-gpt-from-official-source.js | description": {
+    "message": "Load GPT from 'securepubads.g.doubleclick.net' for standard integrations from 'pagead2.googlesyndication.com' for limited ads. [Learn more](https://developers.google.com/publisher-ads-audits/reference/audits/loads-gpt-from-official-source).",
```
Comment (jimper) [inline lighthouse-plugin-publisher-ads/messages/en-US.json:431]: or

**Review by jimper — CHANGES_REQUESTED** [review]

File: lighthouse-plugin-publisher-ads/messages/en-US.json [hunk 5]
```diff
@@ -428,7 +428,7 @@
     "description": ""
   },
   "audits/loads-gpt-from-official-source.js | description": {
-    "message": "Load GPT from 'securepubads.g.doubleclick.net' for standard integrations from 'pagead2.googlesyndication.com' for limited ads. [Learn more](https://developers.google.com/publisher-ads-audits/reference/audits/loads-gpt-from-official-source).",
+    "message": "Load GPT from 'securepubads.g.doubleclick.net' for standard integrationsor from 'pagead2.googlesyndication.com' for limited ads. [Learn more](https://developers.google.com/publisher-ads-audits/reference/audits/loads-gpt-from-official-source).",
```
Comment (jimper) [inline lighthouse-plugin-publisher-ads/messages/en-US.json:431]: integrations or

File: lighthouse-plugin-publisher-ads/audits/loads-gpt-from-official-source.js [hunk 6]
```diff
@@ -23,8 +23,8 @@ const UIStrings = {
   title: 'GPT tag is loaded from an official source',
   failureTitle: 'Load GPT from an official source',
   description:
-  'Load GPT from \'securepubads.g.doubleclick.net\' for standard ' +
-  'integrations from \'pagead2.googlesyndication.com\' for limited ads. ' +
+  'Load GPT from \'securepubads.g.doubleclick.net\' for standard integrations' +
```
Comment (jimper) [inline lighthouse-plugin-publisher-ads/audits/loads-gpt-from-official-source.js:26]: missing space after

File: docs/audits/loads-gpt-from-official-source.md [hunk 7]
```diff
@@ -1,24 +1,24 @@
...
-integrations could use
+Ensure that GPT is being loaded from an official source. Standard integrations
+should use:
 
 ```HTML
 <script async src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"></script>
 ```
 
-And integrations with limited ads could use:
+Integrations with limited ads should use:
```
Comment (jimper) [inline docs/audits/loads-gpt-from-official-source.md:21]: Make "limited ads" a link to https://support.google.com/admanager/answer/9882911

**Review by jimper — CHANGES_REQUESTED** [review]

File: docs/audits/loads-gpt-from-official-source.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+from an official source. The browser must open a connection to each new host
+your page references, which can take significant time. By loading GPT from the
+recommended host you ensure that all ad serving requests are made to a single
+domain, reducing the number of connections to open and improving page load
+performance.
+
+## Recommendations
+
+Ensure that GPT is being loaded from an official source. Standard integrations
+should use:
```
Comment (mmisim) [inline docs/audits/loads-gpt-from-official-source.md:15]:
```suggestion
Ensure that GPT is being loaded from an official source.  For standard integrations,
use:
```

File: docs/audits/loads-gpt-from-official-source.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+```HTML
+<script async src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"></script>
+```
+
+Integrations with limited ads should use:
```
Comment (mmisim) [inline docs/audits/loads-gpt-from-official-source.md:21]:
```suggestion
For integrations with limited ads, use:
```

**Review by mmisim — APPROVED** [review]

**Review by jimper — APPROVED** [review]

**Pull request merged**